src/Automation/AutomationSession.cs

```diff
@@ -19,14 +19,14 @@ internal class AutomationSession
         /// <summary>
         /// ctor - Initializes the app in automation mode
         /// </summary>
-        /// <param name="parameters">The parameters from the command line</param>
-        private AutomationSession(CommandParameters parameters)
+        /// <param name="config">A set of configuration options</param>
+        private AutomationSession(Config config)
```

Comment:
It appears to me that you removed all non-test uses of AutomationSession. Any reason to keep it around?

Reply:
On second thought, I'm guessing we'll want to use this when `Scanner.Scan()` is implemented

Reply:
I've moved the disposal into the SnapshotCommand.Execute